notification/twilio/twiml.go

```diff
@@ -195,7 +195,9 @@ func (t *twiMLResponse) sendResponse() {
 	}
 
 	var buf bytes.Buffer
-	io.WriteString(&buf, xml.Header)
+	if _, err := io.WriteString(&buf, xml.Header); err != nil {
```

Comment:
`bytes.Buffer` will never return an error; if out of memory, it will panic internally. The error value should be ignored

Reply:
done.  set to ignore